web/src/components/AttributeList/AttributeList.tsx

```diff
@@ -1,17 +1,30 @@
-import {TSpanFlatAttribute} from '../../types/Span.types';
-import AttributeRow from '../AttributeRow';
+import AttributeRow from 'components/AttributeRow';
+import {IResultAssertions} from 'components/SpanDetail/SpanDetail';
+import {TSpanFlatAttribute} from 'types/Span.types';
 import * as S from './AttributeList.styled';
 
-interface IAttributeListProps {
+interface IProps {
+  assertions?: IResultAssertions;
   attributeList: TSpanFlatAttribute[];
   onCreateAssertion(attribute: TSpanFlatAttribute): void;
 }
 
-const AttributeList: React.FC<IAttributeListProps> = ({attributeList, onCreateAssertion}) => {
+const AttributeList: React.FC<IProps> = ({assertions, attributeList, onCreateAssertion}) => {
+  const onCopy = (value: string) => {
+    navigator.clipboard.writeText(value);
```

Comment:
Take a look at what I did here https://github.com/kubeshop/tracetest/pull/583/files#diff-198edd331eb1e96f1a2636afe0b86500dcece0a7df2abe028ee881404afac599R14 I think it might overlap

Reply:
Yes, I was thinking the same. You can merge. I can solve the conflicts.

Reply:
Done